src/main/java/com/firebase/geofire/GeoQuery.java

```diff
@@ -411,4 +411,17 @@ public synchronized void setRadius(double radius) {
             this.setupQueries();
         }
     }
+
+    /**
+     * Sets the center and radius (in kilometers) of this query, and triggers new events if necessary.
+     * @param radius The new radius value of this query in kilometers
```

Comment:
Can you please also add an `@param` line for `center` before this line?

`* @param center The new center`